File: Sumar/Sumar.c
#include <stdio.h>

int main(){
    int numeroPara = 0;
    int i = 0, suma = 1;
    printf("Introduce un numero: ");
    scanf("%d",&numeroPara);

    for(i=1;i <= numeroPara; i++){
        suma = suma * i;
    }

    printf("Suma Total: %d",suma);

    return 0;
}

File: Sumar/Sumar.java
import java.util.Scanner;
/**
 * Sumar
 */
public class Sumar {

    public static void main(String[] args) {
        int numero = 0;
        Scanner in = new Scanner(System.in);
        System.out.println("Introduce Un Numero");

        numero = in.nextInt();
        int suma = 0;

        for (int i = 0; i <= numero; i++)
            suma += i;   

        System.out.println("El Resultado es: "+ suma);

    }
}